conans/test/functional/command/upload_compression_test.py

```diff
@@ -78,6 +78,8 @@ def upload_only_tgz_if_needed_test(self):
 
         # Check that conans exists on server
         server_paths = self.servers["default"].server_store
+        rev = server_paths.get_last_revision(ref).revision
+        ref = ref.copy_with_rev(rev)
```

Comment:
Is this really necessary for this test? Seems no

Reply:
Well, the server storage is also blocked (with asserts) to get a path to an export folder. 
I can change to check the presence of the "conan_revisions_root" that points to the parent folder of the package.